### PR TITLE
feat(blog): add WWDC 2022 post

### PR DESCRIPTION
**Pull request by hansemannn**
Please review :)

### CODE REVIEW

**Review by caspahouzer — APPROVED** [review]
This looks great

LGTM

**Comment by cb1kenobi** [comment]
Please DO NOT MERGE until I review. I have some interesting bits coming in the next day or so.

**Review by caspahouzer — APPROVED** [review]
Looks very good. Let’s merge it, when all CLI changes are finished

File: posts/2022/wwdc_2022_test_xcode14_ios16.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## Known Issues
+
+- When using `Ti.UI.TabGroup`, the navigation bar / tab bar does not respect proper safe area. This might be a beta issue, but it's being investigated
+
+Missing a known issue? Please report it via [GitHub](https://github.com/tidev/titanium_mobile/issues)
+
+## Roadmap
+
+- TODAY: Be able to target iOS 16 using Xcode 14 using a nightly build
```
[the remark this answers is not in the document]
Comment (hansemannn) [reply posts/2022/wwdc_2022_test_xcode14_ios16.md:46]: Yep

**Review by cb1kenobi — APPROVED** [review]
LGTM!

**Pull request merged**